### PR TITLE
final changes

### PR DESCRIPTION
**Pull request by EphraYeem**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import re
```
Comment (osherdavid) [inline main.py:7]: Make code good.

Comment (osherdavid) [reply main.py:7]: Make OOP

**Review by osherdavid — COMMENTED** [review]
Do.

File: main.py (hunk 1 above)
Comment (Ashal148148) [reply main.py:7]: OOP is the future

**Pull request merged**